src/sidebar/components/test/annotation-test.js

```diff
@@ -461,28 +427,52 @@ describe('Annotation', () => {
 
       assert.isFalse(wrapper.find('AnnotationActionBar').exists());
     });
+  });
 
-    it('should not show annotations if the annotation is a collapsed reply', () => {
-      fakeMetadata.isReply.returns(true);
-      const wrapper = createComponent({ threadIsCollapsed: true });
+  context('annotation thread is collapsed', () => {
+    context('collapsed reply', () => {
+      beforeEach(() => {
+        fakeMetadata.isReply.returns(true);
+      });
 
-      assert.isFalse(wrapper.find('AnnotationActionBar').exists());
+      it('should not render body or footer', () => {
+        const wrapper = createComponent({ threadIsCollapsed: true });
+
+        assert.isFalse(wrapper.find('AnnotationBody').exists());
+        assert.isFalse(wrapper.find('footer').exists());
+      });
+
+      it('should not show actions', () => {
+        const wrapper = createComponent({ threadIsCollapsed: true });
+
+        assert.isFalse(wrapper.find('AnnotationActionBar').exists());
+      });
     });
 
-    it(
-      'should pass a11y checks',
-      checkAccessibility([
-        {
-          content: () => createComponent(),
-        },
-        {
-          name: 'When editing',
-          content: () => {
-            setEditingMode(true);
-            return createComponent();
-          },
-        },
-      ])
-    );
+    context('collapsed top-level annotation', () => {
+      it('should render body and footer', () => {
+        fakeMetadata.isReply.returns(false);
+        const wrapper = createComponent({ threadIsCollapsed: true });
+
+        assert.isTrue(wrapper.find('AnnotationBody').exists());
+        assert.isTrue(wrapper.find('footer').exists());
+      });
+    });
   });
+
+  it(
```

Comment:
Thinking this may be a good time to fill in some axe tests on this file. Wondering if we could do a few more states.  `replyCount` > 0 with `threadIsCollapsed` both true and false